vue/sbc-common-components/src/components/BaseAddress.vue

```diff
@@ -488,6 +492,20 @@ export default class BaseAddress extends Mixins(ValidationMixin, CountriesProvin
   }
 }
 
+// text field labels
+::v-deep .v-label {
+  color: $gray7;
+  font-size: $px-16;
+  font-weight: normal;
+}
+
+// text field inputs
+::v-deep {
+  .v-input input {
+    color: $gray9;
+  }
+}
+
```

Comment:
If these base styles aren't set then the elements inherit whatever Vuetify provides, which is not what our UI designers want. Many (or all, maybe) of our UIs override these styles when they use this BaseAddress component... I think it's better for the component to use the right styles in the first place :)